embedchain/loaders/local_text.py

```diff
@@ -1,10 +1,15 @@
 class LocalTextLoader:
 
-    def load_data(self, content):
+    def load_data(self, content, file_path=None):
```

Comment:
Change this to take just `file_path` or `url` instead of `content`. 
`content` can be populated internally here.
Make sure to refactor any existing call to `load_data`